doc/rst/language/evolution.rst

```diff
@@ -14,6 +14,40 @@ The purpose of this flag is to identify portions of a program that use a
 language or library feature has recently changed meaning or which is
 expected to change meaning in the future.
 
+version 1.32, September 2023
+----------------------------
+
+Version 1.32 deprecates the ``c_string`` type in user interfaces. Please
+replace occurrences of ``c_string`` with ``c_ptrConst(c_char)``. Note that you
+need to ``use`` or ``import`` the ``CTypes`` module.
```

Comment:
I might adjust to:   "... the ``CTypes`` module to have access to ``c_ptrConst`` and ``c_char``. "